akka-docs/rst/scala/code/docs/actor/ActorDocSpec.scala

```diff
@@ -82,6 +82,25 @@ class DemoActorWrapper extends Actor {
   def receive = Actor.emptyBehavior
 }
 
+class ActorWithMessagesWrapper {
+  //#messages-in-companion
+  object MyActor {
+    // It's a good idea to declare here messages supported by the actor
```

Comment:
It's explained in the docs which include the code sample, let's remove this comment then :)